### ISSUE TITLE
bottom left statusbar overlap

### ISSUE THREAD

**Issue by ronbruins** [Webreaper/Damselfly#493]
Sorry for all the new issues, just trying to give some feedback here.

Is it possible to have the tasks running (like writing exif data) not to be on the same spot(bottom left) as where the e.g. xxx selected shows. Right now I am writing a bunch of exif tags 
<img width="423" alt="image" src="https://github.com/Webreaper/Damselfly/assets/43776917/d0fabaaf-070a-491f-8b9d-26cb44facecb">
 but when I  select a new set of images, I want to make sure all relevant images are selected, by the count of selected, but can't see that now, because the exif task is dynamically shown at the same spot, hiding the 'xxx' selected

**Comment by Webreaper** [comment]
Yeah, that makes sense. I could put the status/work stuff in a different place, potentially. 

There's also a bug in MudBlazor which means that when the status bar gets updated, if you're halfway through typing an autocomplete tag, it clears the selection and you have to start again. I'm waiting for them to fix it, but putting the status somewhere else (e.g., a collapsable sidebar panel) would mitigate this.
